activities.json

```diff
@@ -317,7 +317,8 @@
     "mozPositionIssue": 528,
     "org": "W3C",
     "title": "CSS Relational Pseudo-Class (:has())",
-    "url": "https://drafts.csswg.org/selectors/#relational"
+    "url": "https://drafts.csswg.org/selectors/#relational",
+    "webFeaturesID": "has"
```

Comment:
It is totally fine if the name of the key is different in different projects, but I'll note that https://github.com/Devographics/entities/pull/14/files used a key very close to this, differing only by capitalization. If `webFeaturesId` would be an acceptable name here, they would match exactly.